### ISSUE TITLE
Add institutes.md to pipeline template

### ISSUE THREAD

**Issue by edmundmiller** [nf-core/tools#1992]
### Description of feature

For a list of who is utilizing the pipeline.

**Comment by Emiller88** [comment]
Maybe users? Companies could be included as well.

**Comment by ewels** [comment]
Could render these on the website too 🤔 Is YAML acceptable? Thinking about the [contributors YAML file](https://github.com/nf-core/nf-co.re/blob/master/nf-core-contributors.yaml).

**Comment by Emiller88** [comment]
Yeah, I was thinking we might pull the institute info from there and just make it a quick one line.

**Comment by FriederikeHanssen** [comment]
maybe do this together with the institutional yaml file and not have it on a pipeline level. but e.g. just for QBiC have a list of pipelines> rnaseq, sarek, hlatyping etc.

**Comment by Emiller88** [comment]
Posting what I said in slack for visibility:
> Fair, but that also might lead to people just over reporting what they use. They just throw a bunch of stuff they ran once. I also don't know if it should be at the lab level or overall institute.
> _Ran once with the test config_
